crates/ruff_linter/src/checkers/ast/analyze/deferred_scopes.rs

```diff
@@ -312,6 +313,9 @@ pub(crate) fn deferred_scopes(checker: &mut Checker) {
                     &mut diagnostics,
                 );
             }
+            if checker.enabled(Rule::TooManyAttributes) {
```

Comment:
Should the rule name be `too-many-instance-attributes` to match [pylint's](https://pylint.readthedocs.io/en/stable/user_guide/messages/refactor/too-many-instance-attributes.html) name?

Reply:
Done.